changelogs/unreleased/20180301000403161_change_file_name.yml

```diff
@@ -0,0 +1,2 @@
+"Added":
+  - "The new command receives a `NAME` parameter that will be converted to the unreleased file name"
```

Comment:
I think it would be nice to wrap 'new' in backticks (`). 
Example:
> The `new` command receives...

Otherwise it can be mistaken as a common noun

Reply:
makes sense